love/src/components/OLE/Exposure/ExposureAdd.jsx

```diff
@@ -118,6 +109,10 @@ class ExposureAdd extends Component {
     return exposureFlagStateToStyle[flag] ? exposureFlagStateToStyle[flag] : 'unknown';
   }
 
+  /**
+   * Resets the form by resetting the values of MultiSelect components and RichTextEditor component.
```

Comment:
For future reference, we prefer an active voice when writing doc strings. In this case `Reset` instead of `Resets` and in the next sentence `set` instead of `sets`. This also applies to the other doc strings added in this PR. I'll leave it to you if you want to modify this now or not.

Reply:
Aagh I always miss this part 😅️, I had this in mind when doing the changes, but it slippered me through. I'll make the changes and try to don't miss this in the future. Thank you!